homeassistant/components/calendar/services.yaml

```diff
@@ -18,6 +18,12 @@ create_event:
       example: "Meeting to provide technical review for 'Phoenix' design."
       selector:
         text:
+    location:
```

Comment:
I'd say move to the end so its near the end in the developer tools.

Reply:
Moved.